src/main/resources/import.sql

```diff
@@ -167,29 +167,29 @@ INSERT INTO value (value, description, header_id, category_id) VALUES ("max-age"
 INSERT INTO value (value, description, header_id, category_id) VALUES ("s-maxage", "Overrides max-age or the Expires header, but it only applies to shared caches (e.g., proxies) and is ignored by a private cache. Because this header is depened on how the website is being used it's adviced to double check if you really need this.", 10, 3);
 
 # =========== Set-Cookie ===========
-INSERT INTO value (value, description, header_id, category_id) VALUES ("Expires", "It's not recommended to set this because a client can change his time stamp which can keep the sessionn alive alot longer.", 11, 3);
-INSERT INTO value (value, description, header_id, category_id) VALUES ("Max-Age", "Number of seconds until the cookie expires. A zero or negative number will expire the cookie immediately. Older browsers (ie6, ie7, and ie8) do not support max-age. For other browsers, if both (Expires and Max-Age) are set, Max-Age will have precedence.", 11, 1);
-INSERT INTO value (value, description, header_id, category_id) VALUES ("Domain", "Be sure that this domain can be publicy shown.", 11, 3);
-INSERT INTO value (value, description, header_id, category_id) VALUES ("Secure", "A secure cookie will only be sent to the server when a request is made using SSL and the HTTPS protocol. However, confidential or sensitive information should never be stored or transmitted in HTTP Cookies as the entire mechanism is inherently insecure and this doesn't mean that any information is encrypted, for example.", 11, 1);
+INSERT INTO value (value, description, header_id, category_id) VALUES ("Expires", "It's not recommended to configure the header like this. It means that any client is able to change the value of the time stamp which can keep the session alive alot longer.", 11, 3);
```

Comment:
It means that -> it enables the client to change the value of the time stamp which...